### PR TITLE
Added method for getting valid name for certificates

### PR DESCRIPTION
**Pull request by almassapargali**
Please review this. After some research I couldn't find better way for latinising strings. I wanted to add `activesupport`, but then preferred `barbosa` because it's more lightweight.

### CODE REVIEW

**Comment by KrauseFx** [comment]
`valid_name_for` is never called, is it? 

Also, don't you want to put `(` `)` around the parameters?

**Comment by almassapargali** [comment]
Yeah, I was planning to add it here and call from `sigh` and `produce`, not exactly sure what should I do in this case. I mean how I should git latest version of `fastline_core` on them.

File: lib/fastlane_core.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 # Third Party code
 require 'colored'
+require 'babosa'
```
Comment (KrauseFx) [inline lib/fastlane_core.rb]: Shouldn't this be moved to `developer_center_helper.rb`?

**Comment by KrauseFx** [comment]
Oh, sorry, wasn't aware we're in `fastlane_core`. You can run `sudo bundle install && sudo rake install` on your local machine for `fastlane_core`, then switch to `sigh` or `produce` and run `./bin/sigh` from there.

File: lib/fastlane_core.rb (hunk 1 above)
Comment (almassapargali) [inline lib/fastlane_core.rb]: There were no other requires. So I though you were requiring all 3rd party here. Will update now.

**Comment by almassapargali** [comment]
Then, should I include version bumps in PR, or just note in PR that it needs latest version of `fastline_core`? I prefer second option.

**Comment by KrauseFx** [comment]
You don't have to include the version bump. I'll release `sigh` and `fastlane_core` at the same time then.

**Comment by almassapargali** [comment]
Yep, thanks. Will do soon.

**Comment by KrauseFx** [comment]
Looks great, almost ready to merge, please just add the `(` `)` around the method parameter to be be consistent.

**Comment by KrauseFx** [comment]
Thanks @almassapargali :+1:

**Comment by almassapargali** [comment]
👊

**Pull request merged**